libtenzir/test/flatbuffer.cpp

```diff
@@ -50,37 +48,4 @@ TEST(lifetime) {
   CHECK_EQUAL(counter, 1);
 }
 
-namespace {
-
-struct fixture : public fixtures::deterministic_actor_system {
-  fixture() : fixtures::deterministic_actor_system(TENZIR_PP_STRINGIFY(SUITE)) {
-  }
-};
-
-} // namespace
-
-FIXTURE_SCOPE(flatbuffer_fixture, fixture)
-
-TEST(serialization) {
-  auto fbt = flatbuffer<fbs::Type>{};
-  {
-    auto rt = record_type{
-      {"foo", ip_type{}},
-    };
-    auto chunk = chunk::copy(rt);
-    auto maybe_fbt = flatbuffer<fbs::Type>::make(std::move(chunk));
-    REQUIRE_NOERROR(maybe_fbt);
-    fbt = std::move(*maybe_fbt);
-    auto fbt2 = roundtrip(fbt);
-    CHECK_EQUAL(as_bytes(fbt.chunk()), as_bytes(fbt2.chunk()));
-  }
-  auto fbrt = fbt.slice(*fbt->type_as_record_type());
-  auto fbrtf = fbrt.slice(*fbrt->fields()->Get(0));
-  auto fbrtft = fbrtf.slice(*fbrtf->type_nested_root(), *fbrtf->type());
-  auto fbrtft2 = roundtrip(fbrtft);
-  CHECK_EQUAL(as_bytes(fbrtft.chunk()), as_bytes(fbrtft2.chunk()));
-}
```

Comment:
I think we can bring this test back, it used the actor system fixture, but I think it didn't actually need the fixture.